iio/iio_types.h

```diff
@@ -45,6 +45,7 @@
 
```

Comment:
Could you explain in the commit body why it is needed?

Reply:
Sure